Mathlib/Algebra/Category/ModuleCat/Basic.lean

```diff
@@ -453,6 +453,11 @@ def endMulEquiv : End M ≃* (M →ₗ[R] M) where
   left_inv _ := rfl
   right_inv _ := rfl
 
+/-- `ModuleCat.Hom.hom` as an isomorphism of rings. -/
+@[simps!] def endRingEquiv : End M ≃+* (M →ₗ[R] M) where
+  __ := endMulEquiv M
+  map_add' _ _ := rfl
```

Comment:
Should we drop the `endMulEquiv` above?

Reply:
Good idea, deprecated. It's used in representation theory so I need to change three more files.